### DISCUSSION TITLE
Is Sulu 3.0 still planned for 2022?

### DISCUSSION THREAD

**Discussion by Vandaahl** [sulu/sulu#6854] [General]
I read here that 3.0 was planned for this year: https://sulu.io/blog/looking-ahead-to-2022-we-are-grateful-to-the-sulu-community

Is this still the case?

**Comment by alexander-schranz** [comment]
Hello @Vandaahl,

we decided to change the plans and not doing any BC Breaks this year. Instead we will publish a 2.6 Version with new features for the User. Still we will keep Sulu always uptodate with latest versions of Symfony and PHP. So the release of 3.0 is moved to the next year.

Anything specific what you are looking forward in 3.0?

**Reply by remdan** [reply]
can you also explain the current status in connection with this point? thanks...

https://github.com/sulu/SuluArticleBundle/issues/545

I am waiting for SuluArticleBundle without Elasticsearch...

**Reply by Vandaahl** [reply]
What I’m looking forward to is the new content storage tool. 

Since I’m replying almost a year later, I have to ask: is 3.0 still planned for 2023? 😉

**Reply by ricardoee** [reply]
Or indeed 2025 now?

**Reply by alexander-schranz** [reply]
Thx for the interest in Sulu 3.0. After the tests of the alpha and the great feedback of our partners we scheduled the release to provide the quality our community deserve: https://sulu.io/blog/sulu-3-0-taking-a-bit-more-time-to-get-it-right
